.github/ISSUE_TEMPLATE/breaking_change.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Breaking Change
+about: Track plans for a breaking change
+title: "Removal of ? in v#.#"
+labels: "breaking change"
+assignees: ""
+---
+
+## Broken API or pattern
+
+<!-- Name the APIs involved specifically, etc. -->
```

Comment:
Recommendation for format here? Commented-out examples for these sections (expected to be removed) might be useful.